ElixrTalentConnect/Login/LoginViewController.swift

```diff
@@ -0,0 +1,162 @@
+//
+// LoginViewController.swift
+//  ElixrTalentConnect
+//
+//  Created by Devasurya on 10/01/24.
+//
+
+import UIKit
+import LocalAuthentication
+/// LoginViewcontroller contains  user authentication,sign-in ,sign-up.
+/// View in MVVM.
+class LoginViewController: UIViewController {
+    
+    /// Variables and constants  declaration
+    public  let userInfoPrompt = "Don't have an account? Sign up"
+    public  let textToChangeColor = " Sign up"
+    private var viewModel : LoginViewModel!
+    
+    /// Referencing  Outlets.
+    @IBOutlet weak var outerView: UIView!
+    @IBOutlet weak var signInButton: UIButton!
+    @IBOutlet weak var signInPromptLabel: UILabel!
+    @IBOutlet weak var emailField: UITextField!
+    @IBOutlet weak var passwordField: UITextField!
+    @IBOutlet weak var backGroundImage: UIImageView!
+    
+    /// View Life cycle.- Contain method calls to setup basic UI.
+    /// Setupcode to perform same action as that of IQkeyboard manager.
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        UISetup()
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide), name: UIResponder.keyboardWillHideNotification, object: nil)
+        viewModel = LoginViewModel()
+        self.navigationItem.setHidesBackButton(true, animated: true)
+        
+    }
+    
+    /// SignIntapped is the IBaction of the button "signInButton", which trigers alert actions,validation
+    /// - Parameter sender: UIbutton named "signInButton".
+    @IBAction func SignInTapped(_ sender: UIButton){
```

Comment:
please use appropriate camel casing for function and variable names.
```suggestion
    @IBAction func signInTapped(_ sender: UIButton) {
```